extensions/one-time-secret/src/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { Form, ActionPanel, Action, Clipboard, showToast, Toast } from "@raycast/api";
+import got from "got";
+
+type Values = {
+  lifetime: number;
+  recipient: string;
+  passphrase: string;
+  secret: string;
+};
+
+export default function Command() {
+  async function handleSubmit(values: Values) {
+    console.log(values);
+
+    if (!values.secret) {
+      showToast({
+        style: Toast.Style.Failure,
+        title: "Secret is required",
+      });
+      return;
+    }
+
+    const toast = await showToast({
+      style: Toast.Style.Animated,
+      title: "Storing secret",
+    });
+
+    try {
+      let url = "https://onetimesecret.com/api/v1/share";
+
+      url = `${url}/?secret=${values.secret}`;
+
+      if (values.passphrase) {
+        url = `${url}&passphrase=${values.passphrase}`;
```

Comment:
It would be nice to indicate to the user that this field is optional ;)

Reply:
I've made some changes so that the info for the fields say whether they are required or optional. For the secret field, 

I've added form validation to enforce this. 

I've also added asterisks to the field names of fields that are required.

I've had a look through the docs, but couldn't see another way to mark fields as optional or required, so I hope this is along the lines of what you had in mind (but please let me know if there's a better way!!)